examples/experimental/point-cloud-laz-vr/vr/emulated-vr-display.js

```diff
@@ -0,0 +1,148 @@
+/* global window */
+import {Matrix4} from 'math.gl';
+import {perspectiveFromFieldOfView, fromRotationTranslation} from 'gl-mat4';
+
+const DEGREES_TO_RADIANS = Math.PI / 180;
+const EYE_DISTANCE = 0.03;
+const FOV_DEGREES_Y = 40;
+const FOV_DEGREES_X_MIN = 40;
+const FOV_DEGREES_X_MAX = 40;
+const DEFAULT_ORIENTATION = [0, 0, 0, 1];
+const DEFAULT_POSITION = [0, 0, 0];
+
+export default class EmulatedVRDisplay {
+  constructor() {
+    this.depthFar = 1000;
+    this.depthNear = 0.1;
+    this.layers = [];
+    this.isEmulated = true;
+
+    this.leftEyeParameters = {
+      offset: [-EYE_DISTANCE / 2, 0, 0],
+      fieldOfView: {
+        downDegrees: FOV_DEGREES_Y,
+        leftDegrees: FOV_DEGREES_X_MAX,
+        rightDegrees: FOV_DEGREES_X_MIN,
+        upDegrees: FOV_DEGREES_Y
+      }
+    };
+    this.rightEyeParameters = {
+      offset: [EYE_DISTANCE / 2, 0, 0],
+      fieldOfView: {
+        downDegrees: FOV_DEGREES_Y,
+        leftDegrees: FOV_DEGREES_X_MIN,
+        rightDegrees: FOV_DEGREES_X_MAX,
+        upDegrees: FOV_DEGREES_Y
+      }
+    };
+
+    this.poseMatrix = new Matrix4();
+
+    this._leftEyeMatrix = this._getEyeMatrix(this.leftEyeParameters);
+    this._rightEyeMatrix = this._getEyeMatrix(this.rightEyeParameters);
+  }
+
+  getEyeParameters(whichEye) {
+    const viewport = this._getViewportSize();
+
+    if (whichEye === 'right') {
+      return {...this.rightEyeParameters, ...viewport};
+    } else {
+      return {...this.leftEyeParameters, ...viewport};
+    }
+  }
+
+  getFrameData(vrFrameData) {
+    if (!vrFrameData) {
+      throw new Error('must supply a frameData object');
+    }
+
+    const timestamp = Date.now();
+
+    const leftProjectionMatrix = new Matrix4();
+    const rightProjectionMatrix = new Matrix4();
+    const leftViewMatrix = new Matrix4();
+    const rightViewMatrix = new Matrix4();
+
+    perspectiveFromFieldOfView(leftProjectionMatrix, this.leftEyeParameters.fieldOfView, this.depthNear, this.depthFar);
```

Comment:
@sidx1024 could you try eslint this file, several lines are too long and will fail the eslint test before when checking in.